plugins/scripts/inject-config.js

```diff
@@ -2,6 +2,27 @@
 const fs = require('fs');
 const path = require('path');
 
+function createJsonWithDefaultValues(optimoveCredentials, optimoveMobileCredentials , inAppConsentStrategy) {
```

Comment:
I was thinking more about this method, and I'm not actually sure we will ever need this because configuration is actually validated earlier in the process.

I think there's the `isValidConfig` function that won't let the resources be generated if the JSON wasn't valid.

So, do we need to keep this or is it better to refactor it out?

Reply:
hmm I think I added this method to handle null or undefined credentials, which are legal if just one them is undefined, this method just "making sure" that the Java side will get an empty string instead of null or undefined

Reply:
so I think it's necessary but maybe there is a better way of doing it, what do you think?